### PR TITLE
pythonPackages.pyvis: init at 2021-04-29

### PR DESCRIPTION
**Pull request by erictapen**
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 127811` at 152d326c run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.pyvis</li>
    <li>python39Packages.pyvis</li>
  </ul>
</details>

**Comment by ethancedwards8** [comment]
Result of `nixpkgs-review pr 127811` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.pyvis</li>
    <li>python39Packages.pyvis</li>
  </ul>
</details>
~/.cache/nixp

File: pkgs/development/python-modules/pyvis/default.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+, buildPythonPackage
+, networkx
+, jinja2
+, ipython
+, jsonpickle
+, numpy }:
+
+buildPythonPackage rec {
+  pname = "pyvis";
+  version = "2021-04-29";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pyvis/default.nix:14]:
```suggestion
  version = "unstable-2021-04-29";
```

File: pkgs/development/python-modules/pyvis/default.nix [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    owner = "WestHealth";
+    repo = pname;
+    rev = "4c521302abf9725dcbe7f59962baf85360b2718d";
+    sha256 = "sha256-DYbHQpxtSfiLbzmBGclym/ChM0WLBzSEC/3phDTtGY8=";
+  };
+
+  patches = [
+    # Remove when https://github.com/WestHealth/pyvis/pull/101 is merged.
+    (fetchpatch {
+      url = "https://github.com/WestHealth/pyvis/pull/101/commits/158a34de45f970b17ffd746c6e705b89128e2445.patch";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pyvis/default.nix:28]:
```suggestion
      url = "https://github.com/WestHealth/pyvis/commit/158a34de45f970b17ffd746c6e705b89128e2445.patch";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/pyvis/default.nix (hunk 1 above)
Comment (erictapen) [reply pkgs/development/python-modules/pyvis/default.nix]: Done.

File: pkgs/development/python-modules/pyvis/default.nix (hunk 2 above)
Comment (erictapen) [reply pkgs/development/python-modules/pyvis/default.nix]: Done.

File: pkgs/development/python-modules/pyvis/default.nix [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+, jinja2
+, ipython
+, jsonpickle
+, numpy }:
+
+buildPythonPackage rec {
+  pname = "pyvis";
+  version = "unstable-2021-04-29";
+
+  disabled = !isPy3k;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pyvis/default.nix:16]:
```suggestion
```
no longer required since the package split of python2 and python3.

**Pull request merged**